tests/image_test.rs

```diff
@@ -27,6 +27,41 @@ async fn create_image_test(docker: Docker) -> Result<(), Error> {
     Ok(())
 }
 
+async fn create_image_wasm_test(docker: Docker) -> Result<(), Error> {
+    let options = CreateImageOptions {
+        from_src: "-", // from_src must be "-" when sending the archive in the request body
+        repo: "empty-wasm",
+        ..Default::default()
+    };
+
+    let req_body = hyper::body::Body::from({
+        let mut buffer = Vec::new();
+
+        {
+            let mut builder = tar::Builder::new(&mut buffer);
+            let mut header = tar::Header::new_gnu();
+            header.set_path("entrypoint.wasm")?;
+            header.set_size(0);
+            header.set_cksum();
+
+            builder.append_data(&mut header, "entrypoint.wasm", [].as_slice())?;
+            builder.finish()?;
+        }
+
+        buffer
+    });
+
+    docker
+        .create_image(Some(options), Some(req_body), None)
+        .collect::<Vec<_>>()
+        .await
+        .into_iter()
+        .collect::<Result<Vec<_>, bollard::errors::Error>>()
+        .unwrap();
+
```

Comment:
Could we assert that the image was built by using `inspect_image`?

Reply:
Done